### PR TITLE
add sriov-cni okd mirroring

### PR DESCRIPTION
**Pull request by SchSeba**

### CODE REVIEW

**Comment by sshnaidm** [comment]
thanks @SchSeba

File: core-services/image-mirroring/openshift/_config.yaml [hunk 1]
```diff
@@ -40,9 +40,25 @@ images:
   - "4.17"
   - "4.17.0"
   - latest
+  "sriov-4.17":
```
Comment (zeeke) [inline core-services/image-mirroring/openshift/_config.yaml:43]: why do we need these `sriov` specific tags?

Comment (SchSeba) [reply core-services/image-mirroring/openshift/_config.yaml:43]: because we can't have this as part of the 4.17 ImageStream as it's core OCP payload

**Comment by SchSeba** [comment]
/close
https://github.com/openshift/release/pull/52216

**Pull request closed without merging**